mlrun/artifacts/document.py

```diff
@@ -47,6 +47,7 @@ def __init__(
         self,
         loader_class_name: str = "langchain_community.document_loaders.TextLoader",
         src_name: str = "file_path",
+        mlrun_download_object: bool = False,
```

Comment:
Now that I look at it, I actually think we can just call it `download_object` without the `mlrun` prefix. The reason I wanted the prefix was to separate this parameter (which is handled by MLRun) from the parameters passed to the underlying loader, since I thought we're using `**kwargs` to pass arbitrary params. However, since the loader params are anyway passed in a separate dictionary (the `kwargs` dict) then this extra separation is not needed.
Alternatively, maybe instead of a `kwargs` dict we want to use `**kwargs` and the user will just provide the loader params as 1st-level parameters to the class?